Makefile

```
@@ -407,6 +407,9 @@ test-e2e-local-migration: $(GINKGO)
 test-e2e-local: $(GINKGO)
 	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter="default"
 
+test-e2e-local-ha: $(GINKGO)
+	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter "default || (high-availability && $(HA_MODE))"
```

Should we maybe not run the `default` tests anymore for now? After https://github.com/gardener/gardener/pull/6914, there is not much benefit anymore. The only special thing happening on a multi-zonal seed is the zone pinning, but we probably should create a dedicated test for this only (can also be done in a separate PR).

```suggestion
	./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter "high-availability && $(HA_MODE)"
```

---

Okay @rfranzke 
But we can include simple shoot test + HA upgrade test.
`./hack/test-e2e-local.sh --procs=$(PARALLEL_E2E_TESTS) --label-filter "simple || (high-availability && $(HA_MODE))"`

WDYT ?

---

I would probably just create a new test. The "simple" test is quite heavyweight because it has multiple worker pools and tests the Kubernetes version upgrades. A new test just for creating a small shoot which ensures the zone pinning works as expected is good enough.

---

We have to ensure zero downtime while Kubernetes version upgrades. 
Shall we proceed with same simple test or new shoot update test with single worker ?

---

OK, fair enough, then let's use this existing simple test and maybe enhance it to ensure the zone pinning works as expected.